test/integration/apiserver/admissionwebhook/admission_test.go

```diff
@@ -689,7 +689,9 @@ func testResourceUpdate(c *testContext) {
 		if err != nil {
 			return err
 		}
-		obj.SetAnnotations(map[string]string{"update": "true"})
+		annotations := obj.GetAnnotations()
```

Comment:
If somebody had a buggy mutating webhook that did something like this, it will start failing on upgrade to 1.26.

I would consider this WAI: we don't want users to disable tracking with finalizers.

Reply:
Changed the behavior to always add the annotation back